src/command.c

```diff
@@ -672,6 +674,25 @@ int cmd_quit(struct command *cmd)
     return CORVUS_OK;
 }
 
+int cmd_select(struct command *cmd, struct redis_data *data)
+{
+    ASSERT_TYPE(data, REP_ARRAY);
+    ASSERT_ELEMENTS(data->elements == 2, data);
+
+    struct redis_data *db_data = &data->element[1];
```

Comment:
The second argument of `SELECT` is the index of database. `db_data` is not quite suitable here.